bin/texvcjs

```diff
@@ -12,6 +12,7 @@ program
     .usage('[options] <tex input>')
     .option('--rebuild', 'Rebuild PEGjs grammar before parsing')
     .option('-v, --verbose', 'Show verbose error information')
+    .option('--usemathrm ', 'Use \\mathrm instead of \\mbox to escape some text literals')
```

Comment:
That's not an easy question ... probably because I accidently hit the space-bar ;-)